src/workspace-tree/bazel_package_tree_item.ts

```diff
@@ -78,8 +78,8 @@ export class BazelPackageTreeItem
     return this.packagePath.substring(this.parentPackagePath.length + 1);
   }
 
-  public getIcon(): vscode.ThemeIcon {
-    return vscode.ThemeIcon.Folder;
+  public getIconName(): undefined {
+    return undefined;
   }
```

Comment:
why?

Reply:
At first I was going to prevent the type checker and convert the type `IconName | vscode.ThemeIcon` back in the `BazelWorkspaceTreeProvider`, thus moved the logic there. If we want to get the icon path in `BazekTargetTreeItem` then we have to pass the resource class into that as well (maybe it's actually making more sense?).

Reply:
PTAL